### DISCUSSION TITLE
Issue with Accessing localhost:8083 after Setting Up Docker Container

### DISCUSSION THREAD

**Discussion by tayllana** [community/community#118329] [Programming Help]
I used the following steps to create my Docker container:

Create the following Dockerfile in the project root:
```
FROM php:5.6-apache
COPY . /var/www/html
EXPOSE 80
```
To create the container, navigate to the root directory of the project using VSCode or the terminal:
```
sudo docker build -t php5.6-apache .
sudo docker run -d -p 8083:80 --name www -v /home/tayllana/Docker-containers/www:/var/www/html php5.6-apache
```
To enter the container:
`sudo docker exec -it www /bin/bash`
Execute the following commands inside the container:
```
a2enmod rewrite
service apache2 restart
```
To start the container:
`sudo docker start www`
However, when I access the address localhost:8080, I get the following error:
```
Forbidden
You don't have permission to access / on this server.
Apache/2.4.25 (Debian) Server at localhost Port 8083
```
How do I resolve this?

**Comment by Codwizer** [comment]
The problem you described is related to permissions to access the /var/www/html directory inside your Docker container. 

To fix this "Forbidden" error, you need to set the correct permissions for the files and directories that you copied inside the container when creating it.

To solve this problem, you can follow these steps:

1. Log into the container using the command:
```
sudo docker exec -it www /bin/bash
```

2. Make sure that the correct permissions are set for the /var/www/html directory:
 ```
 chown -R www-data:www-data /var/www/html
 ```

3. After setting the correct permissions, restart Apache inside the container:
```
service apache2 restart
```

After completing these steps, try contacting localhost:8080 again and check if the "Forbidden" problem has been resolved.
